packages/charm/test/cell-operations.test.ts

```diff
@@ -0,0 +1,198 @@
+import { assertEquals, assertRejects } from "@std/assert";
```

Comment:
I think we should just drop these tests, superfluous. We should prefer to extend the `integration.sh` script for more robust testing.